### ISSUE TITLE
Not all bars in waterfall chart is focusable

### ISSUE THREAD

**Issue by pvsleeper** [highcharts/highcharts#16788]
#### Expected behaviour
All the bars in a waterfall chart should be focusable when a11y is enabled. 

#### Actual behaviour
Certain bars in a waterfall chart are not focusable. This seems to happen when the bars have negative values but the axis doesn't have negative values.

#### Live demo with steps to reproduce
https://jsfiddle.net/oehu0tx6/24/

#### Product version
9.3.2

#### Affected browser(s)
Chrome, Egde

**Comment by raf18seb** [comment]
Hi @pvsleeper, thank you for reporting a bug.

**Comment by oysteinmoseng** [comment]
Internal note: Happens because the waterfall point in question has `point.isInside === false`.

**Comment by raf18seb** [comment]
Thanks @oysteinmoseng for a clue!

Indeed, the `isPoinsInside` method is fired **before** the `point.plotY` is recalculated in the `waterfall.prototype.translate` method. Before recalculations, the original `-438` value is taken but the waterfall's column obviously has different y and `shapeArgs`.

When fixing: developer should investigate why `isPointinside` is fired before `plotY` recalculations - the order should be changed or the `isInside` flag should be recalculated again like in this workaround: https://jsfiddle.net/BlackLabel/5dkbsawj/

Tagging as a good exercise for beginners and not strictly related to accessibility.
